src/main/java/com/hrznstudio/galacticraft/server/command/GalacticraftCommands.java

```diff
@@ -23,119 +23,251 @@
 import net.minecraft.util.Formatting;
 import net.minecraft.util.math.BlockPos;
-import net.minecraft.util.math.Vec3d;
-import net.minecraft.world.Heightmap;
+import net.minecraft.util.registry.Registry;
+import net.minecraft.util.registry.RegistryKey;
+import net.minecraft.world.dimension.DimensionType;
 
 import java.util.Collection;
+import java.util.HashMap;
+import java.util.UUID;
 import java.util.function.Consumer;
 
 /**
  * @author <a href="https://github.com/StellarHorizons">StellarHorizons</a>
  */
 public class GalacticraftCommands {
 
+    private static final HashMap<UUID,Long> GCR_HOUSTON_TIMERS = new HashMap<>();
+    private static final long GCR_HOUSTON_TIMER_LENGTH = 12L * 1000L; // in milliseconds
+
     public static void register() {
         CommandRegistrationCallback.EVENT.register((commandDispatcher, b) -> {
-            commandDispatcher.register(LiteralArgumentBuilder.<ServerCommandSource>literal("dimensiontp")
+
+            commandDispatcher.register(
+                    LiteralArgumentBuilder.<ServerCommandSource>literal("gcrhouston")
+                    .executes(GalacticraftCommands::teleportToEarth));
+
+            /* This looks convoluted, but it works. Essentially, it registers three branches of the same command.
+             * One as the base, one to also teleport entities, and one to also teleport to a specific position.
+             * This is because the command I added, to teleport to a specific position, breaks when combined with
+             * teleporting multiple non-player entities for some reason. So, I made it where you can pick
+             * teleporting entities OR setting a custom position to go to, but not both :P
+             */
+            LiteralCommandNode<ServerCommandSource> dimensiontp_root = commandDispatcher.register(
+                    LiteralArgumentBuilder.<ServerCommandSource>literal("dimensiontp")
+                    .requires(serverCommandSource -> serverCommandSource.hasPermissionLevel(2))
+                    .then(CommandManager.argument("dimension", DimensionArgumentType.dimension())
+                    .executes(GalacticraftCommands::teleport)));
+            // TODO: either fix this or remove it
+            /* LiteralCommandNode<ServerCommandSource> dimensiontp_entities = commandDispatcher.register(
+                    LiteralArgumentBuilder.<ServerCommandSource>literal("dimensiontp")
+                    .requires(serverCommandSource -> serverCommandSource.hasPermissionLevel(2))
+                    .then(CommandManager.argument("dimension", DimensionArgumentType.dimension())
+                    .then(CommandManager.argument("entities", EntityArgumentType.entities())
+                    .executes(((GalacticraftCommands::teleportMultiple)))))); */
+            LiteralCommandNode<ServerCommandSource> dimensiontp_pos = commandDispatcher.register(
+                    LiteralArgumentBuilder.<ServerCommandSource>literal("dimensiontp")
```

Comment:
Use `CommandManager#literal` instead of `LiteralArgumentBuilder#literal` to avoid explicitly specifying generics